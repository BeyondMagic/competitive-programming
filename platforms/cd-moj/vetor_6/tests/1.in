5
10 5 8 -100 13
3 6 1 100 -12
