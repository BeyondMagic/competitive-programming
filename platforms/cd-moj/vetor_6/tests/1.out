13 11 9 0 1
